src/hooks/simulation/useSendSimulationToAgir.ts

```diff
@@ -0,0 +1,34 @@
+import { SERVER_URL } from '@/constants/urls'
+import { useCurrentSimulation } from '@/publicodes-state'
+import { useMutation } from '@tanstack/react-query'
+import axios from 'axios'
+
+export function useSendSimulationToAgir() {
```

Comment:
As the enpoint is `/export-situation` what do you think of `useExportSituationToAgir` ?